ui/ops/src/routes/ops/security-events/route.js

```diff
@@ -0,0 +1,18 @@
+export default [
+  {
+    path: 'security-events',
+    components: {
+      default: () => import('./SecurityEventsTable.vue'),
+      sidebar: () => import('./SecurityEventsSideBar.vue')
+    },
+    props: {
+      default: true,
+      sidebar: false
+    },
+    meta: {
+      authentication: {
+        rolesRequired: ['superAdmin', 'admin', 'commissioner', 'operator', 'viewer']
```

Comment:
I don't think viewers should be allowed to see security events, though we're going to be changing all this after MR's work anyway so wouldn't worry too much about it.